CONTRIBUTING.md

```diff
@@ -0,0 +1,65 @@
+# Contributing to ci-imgs
+
+We love your input! We want to make contributing to this project as easy and transparent as possible, whether it's:
+
+- Reporting a bug
+- Discussing the current state of the code
+- Submitting a fix
+- Proposing new features
+
+## We Develop with Github
+We use github to host code, to track issues and feature requests, as well as accept pull requests.
+
+## Report a bug or request a feature
+The best way to get in touch is by [opening an issue](https://github.com/rapidsai/ci-imgs/issues/new/choose).
+
+## To make a code contribution
+1. Fork the repo and create your branch from `main`.
+2. Make your changes.
+3. If you've changed APIs, update the documentation.
+4. Make sure your code lints.
```

Comment:
It's easier to rely on CI, as it runs linter checks for us. Can we remove this step for simplicity?